app/code/Magento/Analytics/ReportXml/Query.php

```diff
@@ -77,12 +77,9 @@ public function getConfig()
     }
 
     /**
-     * Specify data which should be serialized to JSON
-     * @link http://php.net/manual/en/jsonserializable.jsonserialize.php
-     * @return mixed data which can be serialized by <b>json_encode</b>,
-     * which is a value of any type other than a resource.
+     * @inheritDoc
      */
-    public function jsonSerialize()
+    public function jsonSerialize(): array
```

Comment:
```suggestion
    #[\ReturnTypeWillChange]
    public function jsonSerialize()
```

Reply:
Hello @ihor-sviziev 
This is really true.
This is big BIC IMHO.
We fixed this changes.
@karyna-tsymbal-atwix thank you for the fast fix.